### PR TITLE
#1032 - update imputation flag default as auto

### PR DESCRIPTION
**Pull request by ClaudiaCarlucci**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x ] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [ x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [ x] Update to all relevant roxygen headers and examples 
- [x ] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [ x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md`
- [x ] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [ x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [ x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done!  Much love to your accomplishment!

### CODE REVIEW

**Comment by ClaudiaCarlucci** [comment]
@bms63 this is the right one. Apologies for the mistake in the other request

File: NEWS.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ values based on user-defined pairing (#944)
 from a BDS dataset, e.g. ADLB. (#943)
 
 ## Updates of Existing Functions
-
```
Comment (bms63) [inline NEWS.md:35]: This isn't a change to function just to the template.  I think we can not include this in the news as it is not super important to the users.  Please remove.

File: man/adsl.Rd [hunk 2]
```diff
@@ -5,7 +5,7 @@
 \alias{adsl}
 \title{Subject Level Analysis Dataset}
 \format{
-An object of class \code{tbl_df} (inherits from \code{tbl}, \code{data.frame}) with 306 rows and 49 columns.
```
Comment (bms63) [inline man/adsl.Rd:8]: @thomas-neitmann I've seen this happen in another PR.  Is this worrisome?

File: man/reexports.Rd [hunk 3]
```diff
@@ -15,6 +15,6 @@ below to see their documentation.
 \describe{
   \item{dplyr}{\code{\link[dplyr]{desc}}, \code{\link[dplyr]{vars}}}
 
-  \item{rlang}{\code{\link[rlang]{exprs}}}
```
Comment (bms63) [inline man/reexports.Rd:18]: @thomas-neitmann I've seen this happen in another PR. Is this worrisome?

**Review by bms63 — CHANGES_REQUESTED** [review]
Hi Claudia.  Thanks for doing this!  I think we can remove this from news.  I want @thomas-neitmann to comment on these changes to the .Rd files as they are not related to the adeg templates.  Not sure why they are changing.

**Comment by bms63** [comment]
Hey Claudia.  Are you able to finish this PR today?  We have fixed a lot of the issues with the workflows/actions so hopefully my previous comments are not an issue anymore.  Please fix merge conflicts.

**Comment by bms63** [comment]
Hi Claudia, I fixed merged conflicts.

**Comment by bms63** [comment]
Unsure why so many .Rd files needed to get updated.  Can you look through them and see ift hey are just fomratting issues. thanks

**Comment by ClaudiaCarlucci** [comment]
It's right, it's just formatting issues, nothing changed. Please let me know how I can proceed. Thanks

**Comment by bms63** [comment]
Please fix merge conflict and then I can re-approve to get the check_templates firing again.

**Comment by ClaudiaCarlucci** [comment]
Hi @bms63 @millerg23 , It seems all checks went through. The merge is still blocked as Ben you requested some changes, is that okay now?

**Review by millerg23 — APPROVED** [review]
I will approve and see what happens, all looks good @ClaudiaCarlucci.
If not then Ben can approve and merge when he logs on later.

**Pull request merged**